frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx

```diff
@@ -25,13 +24,12 @@ function Inventory() {
         }`}
       >
         <section className="openBag">
-          <img src={OpenBag} alt="Open Bag" className="insideBag" />
           <button
             type="button"
             className="inventoryPopUp__closeButton"
             onClick={handleInventoryToggle}
           >
-            Fermé
+            X
```

Comment:
An aria-label is needed to be more explicit about the button's function.

Reply:
Corrected